_fragments is referenced but not defined or used

**Avogadro version: (please complete the following information from the About box):**
 - Avogadrolibs: 1.99.0
 - Qt: 5.15.10

**Desktop version: (please complete the following information):**
 - OS: Linux
 - Version debian unstable
 - Compiler gcc 13.2.0

**Describe the bug**
Avogadrolibs: 1.99.0 has added fragments from https://github.com/openchemistry/fragments processed in avogadro/qtplugins/templatetool/CMakeLists.txt

The CMakeLists.txt pulls the data from the git repo if `${_fragments}` does not exist. But `_fragments` is not defined anywhere, and more to the point there is no handling in place if it does exist.  This means if it does exists, the install step
```
install(DIRECTORY "${AvogadroLibs_SOURCE_DIR}/../fragments"
```
will fail, unless  `_fragments` happens to be set to `${AvogadroLibs_SOURCE_DIR}/../fragments`.

This is a problem for debian builds, for instance, where fragments are kept in `debian/data/fragments` (via git subtree, since by Debian policy data cannot be pulled from the internet at build time). In this case git clone should not be needed (the data has already been cloned and is available locally).

**To Reproduce**
Steps to reproduce the behavior:
1. `git clone https://github.com/openchemistry/fragments /tmp/fragments`
2. `mkdir build;  cd build`
3. `cmake -D_fragments=/tmp/fragments -DUSE_MMTF=OFF -DUSE_LIBMSYM=OFF ..`
4. See error

**Expected behavior**
Since `_fragment` is referenced in `avogadro/qtplugins/templatetool/CMakeLists.txt`,  you'd expect it can be used to identify the location of the fragments dir, without required git.

**Screenshots**
```
[ 98%] Creating directories for 'fragments'
cd /projects/avogadrolibs/obj-x86_64-linux-gnu/avogadro/qtplugins/templatetool && /usr/bin/cmake -Dcfgdir= -P /projects/avogadrolibs/obj-x86_64-linux-gnu/avogadro/qtplugins/templatetool/fragments-prefix/tmp/fragments-mkdirs.cmake
cd /projects/avogadrolibs/obj-x86_64-linux-gnu/avogadro/qtplugins/templatetool && /usr/bin/cmake -E touch /projects/avogadrolibs/obj-x86_64-linux-gnu/avogadro/qtplugins/templatetool/fragments-prefix/src/fragments-stamp/fragments-mkdir
[ 98%] Performing download step (git clone) for 'fragments'
cd /projects && /usr/bin/cmake -P /projects/avogadrolibs/obj-x86_64-linux-gnu/avogadro/qtplugins/templatetool/fragments-prefix/tmp/fragments-gitclone.cmake
Cloning into 'fragments'...
fatal: unable to access 'https://github.com/openchemistry/fragments/': Could not resolve host: github.com
Cloning into 'fragments'...
fatal: unable to access 'https://github.com/openchemistry/fragments/': Could not resolve host: github.com
Cloning into 'fragments'...
fatal: unable to access 'https://github.com/openchemistry/fragments/': Could not resolve host: github.com
-- Had to git clone more than once: 3 times.
CMake Error at avogadrolibs/obj-x86_64-linux-gnu/avogadro/qtplugins/templatetool/fragments-prefix/tmp/fragments-gitclone.cmake:39 (message):
  Failed to clone repository: 'https://github.com/openchemistry/fragments'


make[3]: *** [avogadro/qtplugins/templatetool/CMakeFiles/fragments.dir/build.make:104: avogadro/qtplugins/templatetool/fragments-prefix/src/fragments-stamp/fragments-download] Error 1
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -29,6 +29,11 @@ set(AvogadroLibs_VERSION
 find_package(Git)
 determine_version(${AvogadroLibs_SOURCE_DIR} ${GIT_EXECUTABLE} "AvogadroLibs")
 
+# path for data / fragment files which could be overriden by package maintainers
+# by default we assume we're using the openchemistry superbuild layout
+# .. this is used by the fragment library, molecules, crystals, etc.
+set(AvogadroLibs_SOURCEDATA_DIR "${AvogadroLibs_SOURCE_DIR}/..")
+
 option(BUILD_SHARED_LIBS "Build with shared libraries" ON)
 
 # Before any plugins are defined, and before any add_subdirectory calls:

--- a/avogadro/qtplugins/insertfragment/CMakeLists.txt
+++ b/avogadro/qtplugins/insertfragment/CMakeLists.txt
@@ -10,8 +10,8 @@ avogadro_plugin(InsertFragment
 
 # Install the fragments & crystals
 # TODO - make this a for loop
-set(_molecules "${AvogadroLibs_SOURCE_DIR}/../molecules")
-set(_crystals "${AvogadroLibs_SOURCE_DIR}/../crystals")
+set(_molecules "${AvogadroLibs_SOURCEDATA_DIR}/molecules")
+set(_crystals "${AvogadroLibs_SOURCEDATA_DIR}/crystals")
 
 # Look in parallel directory for the molecule fragment repository
 if(NOT EXISTS "${_molecules}")
@@ -19,32 +19,32 @@ if(NOT EXISTS "${_molecules}")
 ExternalProject_Add(molecules
     GIT_REPOSITORY https://github.com/openchemistry/molecules
     # or https://github.com/OpenChemistry/molecules/archive/refs/heads/master.zip
-    SOURCE_DIR "${AvogadroLibs_SOURCE_DIR}/../molecules"
+    SOURCE_DIR "${AvogadroLibs_SOURCEDATA_DIR}/molecules"
     CONFIGURE_COMMAND ""
     BUILD_COMMAND ""
     INSTALL_COMMAND ""
 )
 endif()
 
-install(DIRECTORY "${AvogadroLibs_SOURCE_DIR}/../molecules"
+install(DIRECTORY "${AvogadroLibs_SOURCEDATA_DIR}/molecules"
   DESTINATION "${INSTALL_DATA_DIR}/avogadro2"
   PATTERN ".git" EXCLUDE
 )
 
 # crystals
 if(NOT EXISTS "${_crystals}")
-# download molecules... 
+# download molecules...
 ExternalProject_Add(crystals
     GIT_REPOSITORY https://github.com/openchemistry/crystals
     # or https://github.com/OpenChemistry/crystals/archive/refs/heads/master.zip
-    SOURCE_DIR "${AvogadroLibs_SOURCE_DIR}/../crystals"
+    SOURCE_DIR "${AvogadroLibs_SOURCEDATA_DIR}/crystals"
     CONFIGURE_COMMAND ""
     BUILD_COMMAND ""
     INSTALL_COMMAND ""
 )
 endif()
 
-install(DIRECTORY "${AvogadroLibs_SOURCE_DIR}/../crystals"
+install(DIRECTORY "${AvogadroLibs_SOURCEDATA_DIR}/crystals"
 DESTINATION "${INSTALL_DATA_DIR}/avogadro2"
 PATTERN ".git" EXCLUDE
 )

--- a/avogadro/qtplugins/templatetool/CMakeLists.txt
+++ b/avogadro/qtplugins/templatetool/CMakeLists.txt
@@ -24,7 +24,7 @@ avogadro_plugin(TemplateTool
 )
 
 # Install the fragments
-set(_fragments "${AvogadroLibs_SOURCE_DIR}/../fragments")
+set(_fragments "${AvogadroLibs_SOURCEDATA_DIR}/fragments")
 
 # Look in parallel directory for the molecule fragment repository
 if(NOT EXISTS "${_fragments}")
@@ -33,14 +33,14 @@ ExternalProject_Add(fragments
     GIT_REPOSITORY https://github.com/openchemistry/fragments
     GIT_TAG main
     # or https://github.com/OpenChemistry/molecules/archive/refs/heads/master.zip
-    SOURCE_DIR "${AvogadroLibs_SOURCE_DIR}/../fragments"
+    SOURCE_DIR "${AvogadroLibs_SOURCEDATA_DIR}/fragments"
     CONFIGURE_COMMAND ""
     BUILD_COMMAND ""
     INSTALL_COMMAND ""
 )
 endif()
 
-install(DIRECTORY "${AvogadroLibs_SOURCE_DIR}/../fragments"
+install(DIRECTORY "${AvogadroLibs_SOURCEDATA_DIR}/fragments"
   DESTINATION "${INSTALL_DATA_DIR}/avogadro2"
   PATTERN ".git" EXCLUDE
 )

